src/H5FDonion.c

```diff
@@ -361,6 +361,7 @@ H5FDget_onion_revision_count(const char *filename, hid_t fapl_id, size_t *revisi
     herr_t          ret_value    = SUCCEED;
     H5FD_t *        file_drv_ptr = NULL;
     FUNC_ENTER_API(FAIL)
+    H5TRACE3("e", "*si*z", filename, fapl_id, revision_count);
```

Comment:
I feel like this function should probably be called H5FDonion_get_revision_count()

Reply:
I can change it.